src/utils/cookies.js

```diff
@@ -0,0 +1,27 @@
+import firebase from 'firebase/app'
+import 'firebase/analytics'
+
+const hasCookiebot = () => window.Cookiebot
+
+const cookieBotAcceptListener = () => {
+    if (hasCookiebot() && window.Cookiebot.consent?.statistics) {
+        firebase.analytics()
+    }
+}
+
+const cookieBotDeclineListener = () => {
+    if (hasCookiebot() && window.Cookiebot.consent?.statistics) {
```

Comment:
this is crazy /o/ 
what a day we leave in.

Reply:
mistake